pfeffernusse/drivers/__init__.py

```
@@ -13,15 +14,19 @@
 
 available_drivers = [os.path.basename(d).split('_')[0] for d in __all__]
 
-
 def load(label):
-    label = pvl.loads(label)
     for driver in __drivers__:
-        try:
-            try:
-                return driver.get_isd(label)
-            except Exception as e:
-                print(e)
-        except:
-            pass
+        classes = inspect.getmembers(driver, inspect.isclass)
+        for name, obj in classes:
+            
+            if name in ['Framer', 'LineScanner', 'TransverseDistortion', 'RadialDistortion']:
```

There must be a simpler way of determining the callable driver classes.

---

The `if name` is to avoid the mixins. It is totally simpler to use a try/except and just fail over them, but this makes debugging / writing a new driver challenging because the useful errors are lost.

---

I understand that, but there should be a way of finding the usable drivers without maintaining a "not a driver" list.